understanding/20/error-identification.html

```diff
@@ -22,16 +22,17 @@ <h2>In brief</h2>
       <h2>Intent of Error Identification</h2>
       
       
-      <p>The intent of this Success Criterion is to ensure that users are aware that an error
-         has occurred and can determine what is wrong. The error message should be as specific
-         as possible. 
-         In the case of an unsuccessful form submission, re-displaying the form and indicating
-         the fields in error is insufficient for some users to perceive that an error has occurred.
-         Screen reader users, for example, will not know there was an error until they encounter
-         one of the indicators. They may abandon the form altogether before encountering the
-         error indicator, thinking that the page simply is not functional. Per the definition,
-         an "input error" is information provided by the user that is not accepted. This includes: 
+      <p>The intent of this success criterion is to ensure that users are aware that an error
+         has occurred and can determine what is wrong. In the case of an unsuccessful form submission,
+         it is not sufficient to only re-display the form without providing any hint that the submission
+         failed. It also won't be sufficient to merely indicate that fields are in error without providing
+         users with sufficient information about the specific nature of the error.
+         The error messages must be descriptive to help users understand what the problem is.
```

Comment:
I think this could be read as adding to the SC requirement, which I don't think includes "help users understand what the problem is", which is more what "error suggestion" is for. Also, we should probably avoid "must/should/may" language where possible. Suggest removing this line.

Reply:
good points @alastc ... I'll tweak these two aspects

Reply:
Tweaked (moving the text about how descriptive the error should be to refer to 3.3.3)